package.json

```diff
@@ -7,15 +7,18 @@
   },
   "dependencies": {
     "axios": "^0.26.1",
+    "chart.js": "^3.7.1",
```

Comment:
I see three different chart libs, if some of them are not being used then remove the redundant onces. Everything else looks, easy to navigate code.